x-pack/plugins/lens/public/visualizations/xy/color_assignment.ts

```diff
@@ -107,7 +107,7 @@ export function getColorAssignments(
 function getDisabledConfig(accessor: string) {
   return {
     columnId: accessor as string,
-    triggerIcon: 'disabled' as const,
+    triggerIconType: 'disabled' as const,
```

Comment:
If you specify the return type of the function, you don't need to use the **type assertion** (one of the most dangerous operator together with `any`).
Something like this will work correctly:
```
function getDisabledConfig(columnId: string):AccessorConfig {
  return {
    columnId,
    triggerIconType: 'disabled',
  };
}
``` 

The same applies on most changes here

Reply:
Fixed [235e89e](https://github.com/elastic/kibana/pull/147686/commits/235e89ec86bda109bca48c952192c1cf93ab0b44)